models/schema_site_contract.go

```diff
@@ -0,0 +1,30 @@
+package models
+
+type SiteContract struct {
+	Ops   string                 `json:",omitempty"`
+	Path  string                 `json:",omitempty"`
+	Value map[string]interface{} `json:",omitempty"`
+}
+
+func NewSchemaSiteContract(ops, path string, contractRef map[string]interface{}) *SiteContract {
+	contractMap := map[string]interface{}{
+		"contractRef": contractRef,
+	}
+
+	return &SiteContract{
```

Comment:
We moved to a generic version of PatchPayload in the latest code so if you can use this going forward.

```suggestion
	return & PatchPayload{
```